environment.yml

```diff
@@ -1,306 +0,0 @@
-name: sarm
-channels:
```

Comment:
NOTE: I moved the environment.yml into a 2nd branch since it doesn't work currently.  The new one works partially but I've paused that for now.